scripts/start-dev.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+# Call the script with one of either parameter. Ex. ./start-dev.sh frontend
+ENV=$1
+
+if [ -z "$ENV" ]; then
+    echo "Please provide 'frontend' or 'backend' as an argument to the script!"
+    exit 1
+fi
+
+
+if [ $ENV = "frontend" ]; then
+    cd .. && cd frontend
+    npm i
+    npm run start:dev
+elif [ $ENV = "backend"]; then
+    cd .. && cd backend
+    npm i
+    npm run start:dev
+fi
```

Comment:
Support total startup eg.
```suggestion
if [ $ENV = "frontend" ]; then
    cd .. && cd frontend
    npm i
    npm run start:dev
elif [ $ENV = "backend"]; then
    cd .. && cd backend
    npm i
    npm run start:dev
fi
elif [ $ENV = "all" ]; then
    cd .. && cd backend
    npm ci
    npm run start:dev &
    $backend = $!
    cd  .. && cd frontend
    npm ci
    npm run start:dev &
    $frontend = $!
    wait $backend $frontend
```